package.json

```diff
@@ -48,16 +48,12 @@
     "ea-command-ci": "./build.js -run --settings ./settings-test.json --postgresUrl $PG_URL --command",
     "ea-start": "./build.js -run -watch --lint --settings ../ForumCredentials/settings-dev.json --postgresUrlFile ../ForumCredentials/dev-pg-conn.txt",
     "ea-start-xpost-dev": "PORT=4000 ./build.js -run -watch --lint --settings ../ForumCredentials/settings-xpost.json --postgresUrlFile ../ForumCredentials/xpost-pg-conn.txt",
+    "playwright-db": "docker run --rm --name forum-magnum-test-postgres -e POSTGRES_PASSWORD=password -p 5433:5432 -v $PWD/schema/accepted_schema.sql:/docker-entrypoint-initdb.d/00_accepted_schema.sql ankane/pgvector",
+    "playwright-test": "playwright test",
+    "start-playwright": "PORT=3456 ./build.js -run --watch --e2e --settings ./settings-test.json --postgresUrl postgres://postgres:password@localhost:5433/postgres",
     "ea-start-staging-db": "./build.js -run --watch --lint --settings ../ForumCredentials/settings-local-dev-staging-db.json --postgresUrlFile ../ForumCredentials/staging-pg-conn.txt",
     "ea-start-prod-db": "./build.js -run --watch --lint --settings ../ForumCredentials/settings-local-dev-prod-db.json --postgresUrlFile ../ForumCredentials/prod-pg-conn.txt",
-    "ea-start-testing-db": "./build.js -run --watch --cypress --settings ./settings-test.json --postgresUrl ${PG_URL:=$(cat ../ForumCredentials/testing-pg-conn.txt)}",
-    "init-cypress-db": "yarn ea-command-testing 'Globals.dropAndCreatePg({templateId: \"cypress_template\", seed: true, dropExisting: true})'",
-    "init-cypress-db-ci": "yarn exec-accepted-schema && ./scripts/timeout.sh 120 yarn ea-command-ci 'Globals.dropAndCreatePg({templateId: \"cypress_template\", seed: true, dropExisting: true})'",
-    "cypress-run-ci": "yarn wait-on tcp:3000 && yarn ea-cypress-run",
-    "ea-cypress-run": "yarn cypress run --env TESTING_DB_URL=${TESTING_DB_URL:=$(cat ../ForumCredentials/testing-db-conn.txt)},PG_URL=${PG_URL:=$(cat ../ForumCredentials/testing-pg-conn.txt)}",
-    "ea-cypress-open": "yarn cypress open --env TESTING_DB_URL=${TESTING_DB_URL:=$(cat ../ForumCredentials/testing-db-conn.txt)},PG_URL=${PG_URL:=$(cat ../ForumCredentials/testing-pg-conn.txt)}",
-    "lw-cypress-run": "yarn cypress run --env TESTING_DB_URL=${TESTING_DB_URL:=$(cat ../ForumCredentials/testing-db-conn.txt)}}",
-    "lw-cypress-open": "yarn cypress open --env TESTING_DB_URL=${TESTING_DB_URL:=$(cat ../ForumCredentials/testing-db-conn.txt)}}",
+    "ea-start-testing-db": "./build.js -run --watch --e2e --settings ./settings-test.json --postgresUrl ${PG_URL:=$(cat ../ForumCredentials/testing-pg-conn.txt)}",
```

Comment:
Could you call this `ea-start-integration-testing-db` or `ea-start-integration-db`, to indicate that this isn't for playwright

Reply:
Separately, it would be nice to dockerise the database we use for integration tests too (but not for this PR)

Reply:
I'm going to leave it as-is for now for the sake of not creating confusion by renaming things but, as you say above, we should be able to dockerise postgres for integration tests soon which will mean we can remove this command entirely